docs/en/integrations/build-integration.asciidoc

```diff
@@ -0,0 +1,597 @@
+=== Navigation
+
+If an integration has several dashboards, ensure that all of them can be easily navigated.
+To build dashboard navigation use the Markdown visualization type.
+
+For example, the System dashboard provides the following navigation:
+
+// screenshot omitted for now
+
+Source:
+
+[source,text]
+----
+[System Overview](#/dashboard/system-Metrics-system-overview-ecs)  | [Host Overview](#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs) |
+[Containers overview](#/dashboard/system-CPU-slash-Memory-per-container-ecs)
+----
+
+While this can work, it doesn't highlight the selected dashboard.
+Unfortunately the Markdown control is not optimized for navigation,
+which makes it cumbersome to build navigation with highlighted links because each link should be highlighted separately.
+This means that the navigation control you're building has to be cloned as many times as there are dashboard to ensure proper link highlighting. E.g.
+
+[source,text]
+----
+**[System Overview](#/dashboard/system-Metrics-system-overview-ecs)**  | [Host Overview](#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs) |
+[Containers overview](#/dashboard/system-CPU-slash-Memory-per-container-ecs)
+
+[System Overview](#/dashboard/system-Metrics-system-overview-ecs)  | **[Host Overview](#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs)** |
+[Containers overview](#/dashboard/system-CPU-slash-Memory-per-container-ecs)
+
+[System Overview](#/dashboard/system-Metrics-system-overview-ecs)  | [Host Overview](#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs) |
+**[Containers overview](#/dashboard/system-CPU-slash-Memory-per-container-ecs)**
+----
+
+[discrete]
+=== Target system name
+
+Currently we don't make it a rule to show on a dashboard what system it's designed to monitor. The only way to see it is through the dashboard name.
+
+// screenshot omitted for now
+
+When using multiple dashboard on bigger screens, it makes it hard to distinguish between the dashboards. This can be improved by using the Markdown control to show what target system the dashboard is used for.
+
+[discrete]
+=== Naming
+
+Use the following naming convention when building dashboards
+
+[discrete]
+==== Visualizations
+
+[source,text]
+----
+<NAME> [<Metrics | Logs> <PACKAGE NAME>]
+----
+
+Examples:
+
+* Memory Usage Gauge [Metrics System]
+* New groups [Logs System]
+
+Rename all visualizations added to a dashboard to only show the <NAME> part.
+
+// screenshot omitted for now
+
+[discrete]
+==== Dashboards
+
+[source,text]
+----
+[<Metrics | Logs> <PACKAGE NAME>] <Name>
+----
+
+Examples:
+
+* [Metrics System] Host overview
+* [Metrics MongoDB] Overview
+
+[discrete]
+=== Screenshots
+
+Letter casing is important for screenshot descriptions.
+Descriptions are shown in the Kibana UI, so try and keep them clean and consistent.
+
+These descriptions are visualized in the Kibana UI. It would be better experience to have them clean and consistent.
```

Comment:
```suggestion
Descriptions are shown in the {kib} UI, so try and keep them clean and consistent.

These descriptions are visualized in the {kib} UI. It would be better experience to have them clean and consistent.
```